README.md

```diff
@@ -16,6 +16,10 @@
 ### 1. Install plugin
 
 ```shell
+  yarn add eslint-plugin-ember
```

Comment:
This would add it as a dependency instead of a devDependency.

Reply:
Oh oops. I'll change it and update the PR